icdc_schema.py

```diff
@@ -272,8 +272,10 @@ def get_type(self, name):
                             result[HAS_UNIT] = True
                 elif isinstance(prop_desc, list):
                     enum = set()
-                    for t in prop_desc:
-                        if not re.search(r'://', t):
+                    r_url = re.compile(r"://")
+                    url_list = list(filter(r_url.search, prop_desc))
+                    if len(prop_desc) != 1 or len(url_list) != 1:
```

Comment:
It seems to me, if there is only one permissible value, the code won't read it, right?

Reply:
If the url values is the only value in the permissive value, the code will not read it. But if the single permissive value is not the url value, the code will read it.